modules/dynamic-plugins/proc-rhdh-example-external-dynamic-plugins.adoc

```diff
@@ -37,8 +37,8 @@ global:
                       text: Notifications
                     config:
                       pollingIntervalMs: 5000   
-      - package: '@janus-idp/backstage-scaffolder-backend-module-kubernetes-dynamic@1.3.5'
-        # https://registry.npmjs.org/@janus-idp/backstage-scaffolder-backend-module-kubernetes-dynamic
+      - package: '@backstage-community/plugin-scaffolder-backend-module-kubernetes-dynamic@1.3.5'
+        # https://registry.npmjs.org/@backstage-community/plugin-scaffolder-backend-module-kubernetes-dynamic
```

Comment:
```suggestion
        # https://www.npmjs.com/package/@backstage-community/plugin-scaffolder-backend-module-kubernetes
```

Reply:
this package https://www.npmjs.com/package/@backstage-community/plugin-scaffolder-backend-module-kubernetes is not a dynamic plugin IIUC 
how can it work?